scripts/check_coverage.dart

```diff
@@ -0,0 +1,12 @@
+import 'dart:io';
+
+void main(args) {
```

Comment:
Note to reviewer: I was planning on make this all in bash, but it seems that bash can't work with floating point numbers, so I chose dart because dart is already available